usaspending_api/local_settings.example.py

```diff
@@ -0,0 +1,9 @@
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'usaspending_api',
+        'USER': 'usaspending_api',
+        'PASSWORD': '',
+    }
+}
```

Comment:
I think autoenv is a good approach, but I'm definitely on poor footing here and don't practice what I'm preaching 😄